packages/builder/.env.example

```diff
@@ -1,15 +1,18 @@
 # development for local, production for prod builds
 REACT_APP_ENV=development
 
+# removing this will enable the default production or staging chains
+REACT_APP_CHAINS=dev1,dev2
+
 # Bring your own API key
 REACT_APP_ALCHEMY_ID=
 REACT_APP_INFURA_ID=
 REACT_APP_WALLETCONNECT_ID=
 REACT_APP_WALLETCONNECT_PROJECT_ID=
 
-# Get your credentials at https://www.pinata.cloud/
-REACT_APP_PINATA_GATEWAY=
-REACT_APP_PINATA_JWT=
+REACT_APP_IPFS_BASE_URL=http://localhost:3003
+REACT_APP_PINATA_BASE_URL=http://localhost:3003
+REACT_APP_PINATA_JWT=development-token
```

Comment:
We might have to talk about the "default" setup suggestion for first-time contributors. "Bring your own API keys" above would point in the direction of using external services, `localhost:3033` below would suggest we want them to use local services.

Reply:
yes it's true, I think the end solution would be to have everything setup for localdev and explain how people can use external services just in case. wdyt?

Reply:
Sounds good to me.